_pylibczi/Image.h

```diff
@@ -0,0 +1,286 @@
+//
```

Comment:
one way to keep verbose implementation separate from concise "header" is to have this .h file include a .inl file that has all the long inlined implementations.